src/app/inventory/InventoryItem.tsx

```diff
@@ -263,11 +263,11 @@ function selectedSubclassPath(talentGrid: DimTalentGrid) {
   for (const node of talentGrid.nodes) {
     if (node.activated && nodeHashToSubclassPath[node.hash]) {
       const def = nodeHashToSubclassPath[node.hash];
-      const superNode = def.superHash && talentGrid.nodes.find((n) => n.hash === def.superHash);
+      const superNode = talentGrid.nodes.find((n) => n.hash === def?.superHash);
```

Comment:
This is actually shortcutting the whole `find`, but it probably doesn't matter because this isn't run often.

Reply:
some checks didn't make complete sense, in the context of we have complete control of `nodeHashToSubclassPath` so i cleaned it up a bit